package-lock.json

```diff
@@ -2258,7 +2258,7 @@
     },
     "array-equal": {
       "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/array-equal/-/array-equal-1.0.0.tgz",
+      "resolved": "http://registry.npmjs.org/array-equal/-/array-equal-1.0.0.tgz",
```

Comment:
Can you undo the https -> http here?

One way you might be able to force your npm from doing this is:
```
rm -f package-lock.json
rm -rf ~/.npm
npm install --prefer-online
git reset --hard HEAD
npm install
git diff -- package-lock.json
```
Others might have found other solutions to this and should share them if they know!

Reply:
I followed your instructions, and saw no change in package-lock.json, but I did fix up the entries manually.

Reply:
Thanks for trying. Somehow I got to work. I'll keep looking into it as I can.